dlpy/applications/enet.py

```diff
@@ -0,0 +1,271 @@
+        automatically detect the number of classes based on the training set.
+        Default: 2
+    n_channels : int, optional
+        Specifies the number of the channels (i.e., depth) of the input layer.
+        Default: 3
+    width : int, optional
+        Specifies the width of the input layer.
+        Default: 512
+    height : int, optional
+        Specifies the height of the input layer.
+        Default: 512
+    scale : double, optional
+        Specifies a scaling factor to be applied to each pixel intensity values.
+        Default: 1.0/255
+    norm_stds : double or iter-of-doubles, optional
+        Specifies a standard deviation for each channel in the input data.
+        The final input data is normalized with specified means and standard deviations.
+    offsets : double or iter-of-doubles, optional
+        Specifies an offset for each channel in the input data. The final input
+        data is set after applying scaling and subtracting the specified offsets.
+    random_mutation : string, optional
+        Specifies how to apply data augmentations/mutations to the data in the
+        input layer.
+        Valid Values: 'none', 'random'
+    init : str
+        Specifies the initialization scheme for convolution layers.
+        Valid Values: XAVIER, UNIFORM, NORMAL, CAUCHY, XAVIER1, XAVIER2, MSRA, MSRA1, MSRA2
+        Default: None
+    random_flip : string, optional
+        Specifies how to flip the data in the input layer when image data is
+        used. Approximately half of the input data is subject to flipping.
+        Valid Values: 'h', 'hv', 'v', 'none'
+    random_crop : string, optional
+        Specifies how to crop the data in the input layer when image data is
+        used. Images are cropped to the values that are specified in the width
+        and height parameters. Only the images with one or both dimensions
+        that are larger than those sizes are cropped.
+        Valid Values: 'none', 'unique', 'randomresized', 'resizethencrop'
+    output_image_type: string, optional
+        Specifies the output image type of this layer.
+        possible values: [ WIDE, PNG, BASE64 ]
+        default: WIDE
+    output_image_prob: bool, options
+        Does not include probabilities if doing classification (default).
+
+
+    Returns
+    -------
+    :class:`Sequential`
+
+    References
+    ----------
+    https://arxiv.org/abs/1606.02147
+
+    '''
+    parameters = locals()
+    input_parameters = get_layer_options(input_layer_options, parameters)
+    inp = Input(**input_parameters, name='InputLayer_1')
+
+    # initial
+    x = initialBlock(inp)
+
+    # stage one
+    x = downsamplingBNeck(x, 16, 64)
+    for i in range(4):
+        x = regularBNeck(x, 64, 64)
+
+    # stage two
+    x = downsamplingBNeck(x, 64, 128)
+    for i in range(2):
+        x = regularBNeck(x, 128, 128)
+        x = regularBNeck(x, 128, 128)
+    # x = regularBNeck(x, 128, 128)
```

Comment:
please remove this line if we dont need it. if we need it to be commented out, please also explain why it is there but commented

Reply:
Fixed.